tests/scripts/kubeadm.sh

```diff
@@ -50,6 +51,7 @@ install_master(){
     echo "wait for K8s master node to be Ready"
     INC=0
     while [[ $INC -lt 20 ]]; do
+          # shellcheck disable=SC2086 # Save to use single quote
         kube_ready=$(kubectl get node -o jsonpath='{.items['$count'].status.conditions[?(@.reason == "KubeletReady")].status}')
```

Comment:
shellcheck.net doesn't flag this line. What's going on?  Do we need to update our shellcheck version?

https://www.shellcheck.net/

Reply:
it is showing an error for `count` changed that